src/display/lv_display.c

```diff
@@ -1031,6 +1031,37 @@ void lv_display_rotate_area(lv_display_t * disp, lv_area_t * area)
     }
 }
 
+uint32_t lv_display_get_draw_buf_size(lv_display_t * disp)
+{
+    if(!disp) disp = lv_display_get_default();
+    if(!disp) return 0;
+
+    if(disp->buf_1) {
+        return disp->buf_1->data_size;
+    }
+    return 0;
+}
+
+uint32_t lv_display_get_invalidated_draw_buf_size(lv_display_t * disp, uint32_t width, uint32_t height)
+{
+    if(!disp) disp = lv_display_get_default();
+    if(!disp) return 0;
+
+    if(disp->render_mode == LV_DISPLAY_RENDER_MODE_FULL) {
+        return disp->buf_act->data_size;
+    }
+
+    lv_color_format_t cf = lv_display_get_color_format(disp);
+    uint32_t stride = width * lv_color_format_get_bpp(cf);
+    stride = (stride + 7) >> 3; /* Round up, convert to bytes */
```

Comment:
Just a suggestion.

```suggestion
    uint32_t stride = width * lv_color_format_get_size(cf);
```

Reply:
Very much appreciated, thanks :D

Reply:
@liamHowatt can not use this one unfortunately since it returns bytes per pixel and in the  case of I1 its rounded.